src/UnitTests/Program.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+using NUnitLite;
+using System.Reflection;
+
+namespace NodaTime.Test
```

Comment:
The namespace and copyright header would have to be changed. But the change towards an executable makes sense, I wanted to go in this direction as well (for other reasons, unrelated to .Net Core). Thanks!

Reply:
Fixed it. Let me know if it is still unsuitable. :)